internal/deviations/deviations.go

```diff
@@ -597,6 +597,12 @@ func ISISCounterPartChangesUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetIsisCounterPartChangesUnsupported()
 }
 
+// SkipTCPNegotiatedMssCheck returns true for devices that do not
+// support telemetry to check negotiated tcp mss value.
+func SkipTCPNegotiatedMssCheck(dut *ondatra.DUTDevice) bool {
```

Comment:
ugh, sorry I missed this in the last passes, but "Mss" will need to be "MSS"

Reply:
updated.